solutions/hw3/task_overview.md

```diff
@@ -0,0 +1,5 @@
+# Homework 3
+### Preparing data
+### Analyze
+### Visualize
+### Sankey diagram
```

Comment:
You Sankey diagram looks awesome, but I think you need to pre-filter out people from `test` that have no information about survival: otherwise, it's not obvious that only some people are connected to `died`/`survived` and some are not.